.eslintrc.js

```diff
@@ -59,7 +59,7 @@ module.exports = {
     '@typescript-eslint/no-var-requires':         'off',
     'vue/one-component-per-file':                 'off',
     'vue/no-deprecated-slot-attribute':           'off',
-    'vue/require-explicit-emits':                 'off',
+    'vue/require-explicit-emits':                 'error',
```

Comment:
Sorry for being picky, but could we remove this line and set it as errors in `.eslintrc.default.js`, so it's used in all the projects of the monorepo?

Reply:
I updated it.

It looks like the original change still caught everything in the monorepo with `yarn lint`. What scenario are you wanting to cover because I didn't verify anything else.

Reply:
It's important to have separate common rules adopted everywhere and linting rules strictly applied for the rest of the project. `@rancher/components` has its own linting scripts, can't say the same for other `pkg`. There's no need for you to try, since they are already included in CI.